internal/service/federateddatabaseinstance/data_source_federated_database_instance.go

```diff
@@ -229,10 +228,6 @@ func schemaFederatedDatabaseInstanceStoresDataSource() *schema.Schema {
 					Type:     schema.TypeString,
 					Computed: true,
 				},
-				"cluster_id": {
```

Comment:
apart from removing the attribute from the schema, have you removed logic from the CRUD operations?

Reply:
I was searching through and couldn't find any usage of `cluster_id` or `clusterId`